syntax = "proto3";

package WinSearchStore;

service ExplorerService
{
	rpc GetResult(DataRequest) returns (DataResponse);
}

message DataRequest
{
	int32 value = 1;
}

message DataResponse
{
	string message = 1;
}
